modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -61,58 +55,63 @@ Certificate files are typically stored as Base64 files, with extensions such as
 +
 [subs="+attributes,+quotes"]
 ----
 $ {orch-cli} label configmap custom-certs app.kubernetes.io/part-of=che.eclipse.org app.kubernetes.io/component=ca-bundle -n <{project-context}-namespace-name>
 ----
 
-. Deploy {prod-short} if it has not been deployed before. Otherwise wait until the rollout of {prod-short} components finishes. If there are running workspaces, they should be restarted for the changes to take effect.
+. Deploy {prod-short} if it hasn't been deployed before. Otherwise wait until the rollout of {prod-short} components finishes. Restart running workspaces for the changes to take effect.
+
+== Troubleshooting imported certificate issues
+
+If issues occur after adding the certificates, verify the below specified values at a {prod-short} installation and workspace level.
 
-== Verification at the {prod-short} installation level
 
-When something does not work as expected after adding the certificates, here is a list of things to verify:
+.Verifying imported certificates at the {prod-short} installation level
 
-- In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/understanding/olm-what-operators-are.html[Operator] deployment, the namespace where the `CheCluster` is located contains labeled ConfigMaps with the right content:
+* In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/understanding/olm-what-operators-are.html[Operator] deployment, the namespace where the `CheCluster` is located contains labeled ConfigMaps with the right content:
+// Correct content? Relevant content?
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get cm --selector=app.kubernetes.io/component=ca-bundle,app.kubernetes.io/part-of=che.eclipse.org -n {prod-namespace}
 ----
 +
-Check the content of ConfigMap by running:
+Check the content of ConfigMap by entering this command:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get cm __<name>__ -n {prod-namespace} -o yaml
 ----
 
-- {prod-short} Pod Volumes list contains a volume that uses `ca-certs-merged` ConfigMap as data-source.
-To get the list of Volumes of the {prod-short} Pod:
+* {prod-short} Pod Volumes list contains a volume that uses `ca-certs-merged` ConfigMap as data-source.
+To get the list of Volumes of the {prod-short} Pod, run:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get pod -o json __<{prod-id-short}-pod-name>__ -n {prod-namespace} | jq .spec.volumes
 ----
 +
-- {prod-short} mounts certificates in folder `/public-certs/` of the {prod-short} server container. This command returns the list of files in that folder:
+* {prod-short} mounts certificates in the `/public-certs/` folder of the {prod-short} server container. Enter the following command to view the list of files in this folder:
```

Comment:
Reword to be consistent with your previous commands: 

`To view the list of files in this folder, enter:`